src/esp/nav/PathFinder.cpp

```diff
@@ -1400,7 +1400,7 @@ HitRecord PathFinder::closestObstacleSurfacePoint(
 }
 
 bool PathFinder::isNavigable(const vec3f& pt, const float maxYDelta) const {
-  return pimpl_->isNavigable(pt);
+  return pimpl_->isNavigable(pt, maxYDelta);
```

Comment:
i'm late to the party (sorry), but maybe the pimpls shouldn't have any default parameters, in general, and have defaults only in the public api? to avoid this kind of issues